mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ActionButtons.tsx

```diff
@@ -0,0 +1,108 @@
+import { View } from '@components'
+import { IconName, IconType, ThemedIcon, ThemedTextV2, ThemedTouchableOpacityV2 } from '@components/themed'
+import { useFeatureFlagContext } from '@contexts/FeatureFlagContext'
+import { NavigationProp, useNavigation } from '@react-navigation/native'
+import { RootState } from '@store'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { ScrollView, Text } from 'react-native'
+import { useSelector } from 'react-redux'
+import { PortfolioParamList } from '../PortfolioNavigator'
+
+export interface ActionButtonsProps {
+  name: string
+  icon: IconName
+  iconType: IconType
+  onPress: () => void
+  badge?: string | number
+}
+
+export function ActionButtons (): JSX.Element {
+  const { isFeatureAvailable } = useFeatureFlagContext()
+  const navigation = useNavigation<NavigationProp<PortfolioParamList>>()
+  const { futureSwaps } = useSelector((state: RootState) => state.futureSwaps)
+
+  const actions: ActionButtonsProps[] = [
+    {
+      name: translate('components/ActionButtons', 'Send'),
+      icon: 'arrow-up-right',
+      iconType: 'Feather',
+      onPress: () => navigation.navigate('Send')
+    }, {
+      name: translate('components/ActionButtons', 'Receive'),
+      icon: 'arrow-down-left',
+      iconType: 'Feather',
+      onPress: () => navigation.navigate('Receive')
+    }, {
+      name: translate('components/ActionButtons', 'Swap'),
+      icon: 'repeat',
+      iconType: 'Feather',
+      onPress: () => navigation.navigate({
+        name: 'Convert',
+        params: { mode: 'utxosToAccount' },
+        merge: true
+      })
+    }, {
+      name: translate('components/ActionButtons', 'Transactions'),
+      icon: 'calendar',
+      iconType: 'Feather',
+      onPress: () => navigation.navigate('TransactionsScreen')
+    }
+  ]
+
+  if (isFeatureAvailable('future_swap')) {
```

Comment:
i think if there is no future swap, the button will not be display too, according to figma